widgetastic_manageiq.py

```diff
@@ -875,6 +876,27 @@ def first_page(self):
     def last_page(self):
         self.paginator.last_page()
 
+    def pages(self):
+        """Generator to iterate over pages, yielding after moving to the next page"""
+        if self.exists:
+            # start iterating at the first page
+            if self.cur_page != 1:
+                logger.debug('Resetting paginator to first page')
+                self.first_page()
+
+            # Adding 1 to pages_amount to include the last page in loop
+            for page in range(1, self.pages_amount + 1):
```

Comment:
you can use just range(self._pages_amount) since you don't use the number somewhere.